### PR TITLE
ConArm integration for BaseMetals

### PR DESCRIPTION
**Pull request by YaibaToKen**
Adds integration between Construct's Armory and BaseMetals. The Eternal Ice trait (TraitIcy.java) is mostly a stub as I couldn't figure out how to query each Armor part for it's traits.

### CODE REVIEW

File: src/main/java/com/mcmoddev/lib/integration/plugins/tinkers/TinkersMaterial.java [hunk 1]
```diff
@@ -33,6 +37,8 @@
 import slimeknights.tconstruct.library.materials.HeadMaterialStats;
 import slimeknights.tconstruct.library.materials.Material;
 
+import static java.lang.Float.*;
```
Comment (dshadowwolf) [inline src/main/java/com/mcmoddev/lib/integration/plugins/tinkers/TinkersMaterial.java]: Why ? This is pointless - we don't sidestep the boxing/unboxing like this for clarity reasons.

**Review by dshadowwolf — CHANGES_REQUESTED** [review]
Overall solid, but as noted, we do not sidestep boxing/unboxing by using static imports of the class methods.

File: src/main/java/com/mcmoddev/lib/integration/plugins/tinkers/TinkersMaterial.java (hunk 1 above)
Comment (YaibaToKen) [reply src/main/java/com/mcmoddev/lib/integration/plugins/tinkers/TinkersMaterial.java]: Changes have been pushed and should hopefully resolve the requested

**Review by jriwanek — APPROVED** [review]
Looks good now.

**Review by dshadowwolf — APPROVED** [review]
Much better - I'll fix the idiocy I seem to have introduced

**Pull request merged**